### PR TITLE
lambdasoup 0.7.0

### PR DESCRIPTION
**Pull request by dmbaturin**

### CODE REVIEW

**Comment by dmbaturin** [comment]
Unbuildable due to dependencies until #132 is merged.

File: packages/lambdasoup-windows/lambdasoup-windows.0.7.0/opam [hunk 1]
```diff
@@ -0,0 +1,46 @@
+pam-version: "2.0"
```
Comment (toots) [inline packages/lambdasoup-windows/lambdasoup-windows.0.7.0/opam]:
```suggestion
opam-version: "2.0"
```

File: packages/lambdasoup-windows/lambdasoup-windows.0.7.0/opam [hunk 2]
```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+
```
Comment (toots) [inline packages/lambdasoup-windows/lambdasoup-windows.0.7.0/opam]:
```suggestion
```

File: packages/lambdasoup-windows/lambdasoup-windows.0.7.0/opam [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+synopsis: "Easy functional HTML scraping and manipulation with CSS selectors"
+
```
Comment (toots) [inline packages/lambdasoup-windows/lambdasoup-windows.0.7.0/opam]:
```suggestion
```

**Comment by dmbaturin** [comment]
Is there still anything missing to have it merged?

**Comment by toots** [comment]
> Is there still anything missing to have it merged?

Yeah, the CI is still failing.. Looks like it needs `markup-windows`..

**Pull request merged**